crates/uv/tests/lock.rs

```diff
@@ -8203,7 +8203,6 @@ fn lock_change_index() -> Result<()> {
 
     ----- stderr -----
     warning: `uv lock` is experimental and may change without warning
-    Ignoring existing lockfile due to removal of referenced registry: https://pypi-proxy.fly.dev/basic-auth/simple
```

Comment:
I made this all debug. Not sure...

Reply:
I think I might be inclined to show it. Or at least, I feel like we should show something indicating that we did (or tried to) do something to update the lock file. I think Cargo does this.

Reply:
@zanieb -- Do you have an opinion on whether any of these "We can't reuse the lockfile because..." messages should be printed to stderr by default, or as `debug!`?

Reply:
I think the status quo is that we haven't shown anything regarding the lock without verbose enabled, so let's continue with that for now. It seems easy to adjust in response to user feedback and I think a decision will be more obvious once we play with these changes for a bit longer.